app/src/features/Batches/BatchList.tsx

```diff
@@ -130,6 +82,34 @@ const BatchList = (): JSX.Element => {
     }
   );
 
+  const convertDateToTime = (batchStart: string, batchEnd: string) => {
+    const timeBatchStart = new Date(batchStart).getTime();
+    const timeBatchEnd = new Date(batchEnd).getTime();
+    const duration = timeBatchEnd - timeBatchStart;
+    const hours = Math.floor((duration / (1000 * 60 * 60)) % 24);
+    const minutes = Math.floor((duration / (1000 * 60)) % 60);
+    return { hours, minutes };
+  };
+
+  const getBatchDuration = (batch: Batch) => {
+    const batchEnd = batch.completed_at ?? batch.canceled_at;
+    if (batchEnd) {
+      const { hours, minutes } = convertDateToTime(batch.created_at, batchEnd);
+      if (hours > 0 && minutes > 0) {
+        return `${t("hour", { count: hours })} ${t("minute", {
+          count: minutes,
+        })}`;
+      } else if (hours > 0 && minutes === 0) {
+        return t("hour", { count: hours });
+      } else {
+        return t("minute", { count: minutes });
+      }
```

Comment:
I find this to be too complicated. Can't we use `date.toISOString().substr(11, 8);` and use the format HH:MM:SS instead ? then we don't need translation. I think it's clear enough if we use a 🕐 emoji.

Reply:
If it's during more than one days it can't work...